client/elife-theme/src/elements/Steps.js

```diff
@@ -0,0 +1,41 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export default css`
+  margin-bottom: 100px;
+`
+
+const stepCenter = (isCurrent, isPast) => {
+  if (isCurrent) {
+    return '#fff'
+  } else if (isPast) {
+    return th('colorPrimary')
+  }
+  return th('colorFurniture')
+}
+
+export const Step = css`
+  height: 20px;
+  width: 20px;
+  background-color: ${({ isCurrent, isPast }) => stepCenter(isCurrent, isPast)};
+  border-color: ${({ isCurrent, isPast }) =>
+    isCurrent || isPast ? th('colorPrimary') : th('colorFurniture')};
```

Comment:
Oh this is neat. I didn't know you could return an interpolated function from and interpolated function.